### DISCUSSION TITLE
Mangal 4 is coming soon...

### DISCUSSION THREAD

**Discussion by metafates** [metafates/mangal#118] [Announcements]
The most important feature this major version will bring is significantly improved caching mechanism which makes Mangal *extremely* fast and responsive.

Now, mangal makes almost no requests to the servers. This includes Anilist, Custom (lua) sources and built-in sources.

[Live changelog](https://github.com/metafates/mangal/blob/dev/CHANGELOG.md) (This is the **dev** branch)

**Stay tuned 🐳**

**Comment by justchokingaround** [comment]
sheeeeeeeesh
